src/altButton.js

```diff
@@ -0,0 +1,25 @@
+import Button from './button.js';
+
+class AltButton extends Button {
+  constructor(
+    text,
+    width,
+    lang = 'en',
+    altText,
+    code,
```

Comment:
```code,``` here **vs** ```code``` in LetterButton => no comma => keep the same code style

Reply:
Thank you! Fix